frontend/test/__support__/e2e/commands/api/dashboard.js

```diff
@@ -1,9 +1,23 @@
 Cypress.Commands.add(
   "createDashboard",
-  ({ name = "Test Dashboard", ...dashboardDetails } = {}) => {
+  ({
+    name = "Test Dashboard",
+    enable_embedding = false,
+    embedding_params = {},
+    ...dashboardDetails
+  } = {}) => {
     cy.log(`Create a dashboard: ${name}`);
 
     // For all the possible keys, refer to `src/metabase/api/dashboard.clj`
-    return cy.request("POST", "/api/dashboard", { name, ...dashboardDetails });
+    cy.request("POST", "/api/dashboard", { name, ...dashboardDetails }).then(
+      ({ body }) => {
+        if (enable_embedding) {
```

Comment:
A similar approach used in `createQuestion`. For flags that aren't supported in `POST` endpoints, issue a `PUT` immediately.